libs/vkd3d/swapchain.c

```diff
@@ -0,0 +1,2216 @@
+/*
+ * Copyright 2020 Joshua Ashton for Valve Software
```

Comment:
If this is mostly copied from Wine, shouldn't the copyright statements also be included?

Reply:
Resolved locally.

Reply:
Would be nice to have that resolved non-locally as well.

Reply:
It is resolved below.